cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,41 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until database is available."""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        attempts = 0
+        max_attempts = 30
+        wait_time = 1
+
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                db_conn.cursor()
+            except OperationalError:
+                attempts += 1
+                if attempts >= max_attempts:
+                    self.stdout.write(
+                        self.style.ERROR(
+                            "Database unavailable after maximum attempts. "
+                            "Exiting..."
+                        )
+                    )
+                    raise OperationalError(
+                        "Maximum attempts reached. "
+                        "Could not connect to the database."
+                    )
```

Comment:
You are raising an OperationalError here, but it seems like the error is not caught anywhere. This might cause your program to stop execution. It would be better to handle this error and provide a graceful shutdown of the program if the maximum number of attempts is reached.